vowpalwabbit/core/src/reductions/cb/cb_explore_adf_large_action_space.cc

```diff
@@ -493,6 +508,12 @@ VW::LEARNER::base_learner* VW::reductions::cb_explore_adf_large_action_space_set
     options.insert("no_predict", "");
   }
 
+  if (options.was_supplied("squarecb"))
+  {
+    apply_shrink_factor = true;
+    gamma = options.get_typed_option<float>("gamma_scale").value();
```

Comment:
I think gamma needs to be computed like this (see squarecb wiki)?
gamma = gamma_scale * t^{gamma_exponent}

Reply:
do you want to link the wiki for reference?

yes looking at the square cb code it looks like that, will need to also save load the model for that

Reply:
See "Options" section [here](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Contextual-Bandit-Exploration-with-SquareCB#options)